src/main/java/com/bmc/truesight/meter/plugin/remedy/util/PluginTemplateValidator.java

```diff
@@ -45,7 +45,9 @@ public boolean validate(Template template) throws ValidationException {
         // validate payload configuration
         Map<String, String> properties = payload.getProperties();
         for (String key : properties.keySet()) {
-
+        	if(!StringUtil.isValidJavaIdentifier(key)){
+        		throw new ValidationException(StringUtil.format(Constants.PROPERTY_NAME_CONTAINS_SPACE, new Object[]{key.trim()}));
```

Comment:
now that it is  not just space, should we call something else?

`Constants.PROPERTY_NAME_CONTAINS_SPACE`